builtin/logical/totp/path_code.go

```diff
@@ -14,7 +14,7 @@ import (
 
 func pathCode(b *backend) *framework.Path {
 	return &framework.Path{
-		Pattern: "code/" + framework.GenericNameRegex("name"),
+		Pattern: "code/" + framework.GenericNameWithAtRegex("name"),
```

Comment:
Are we likely to see this used elsewhere? Should this really be in the `framework` package?

Reply:
It is unlikely that we see it elsewhere. I originally had it in the totp package itself, but then moved to framework to keep it close to GenericNameRegex to increases chances of changes to that reflected in this as well.